app/assets/stylesheets/dcv/dcv.css.scss

```diff
@@ -927,6 +927,13 @@ h2[itemprop="name"] {
 	#cboxClose {
 		top: 0;
 		background-color: $color_9;
+		font-family: "Glyphicons Halflings";
+	}
+	#cboxDrag {
+		top: 0;
+		right: 27px;
+		background-color: $color_9;
+		font-family: "Glyphicons Halflings";
 		border-bottom-left-radius: 4px;
```

Comment:
may want to add cursor _grab_ and _grabbing_ on _hover_ and _drag_ (perhaps active state or as a js `is being dragged` added class), respectively